proto/viam/service/motion/v1/motion.proto

```diff
@@ -120,9 +120,15 @@ message MoveOnMapResponse {
   bool success = 1;
 }
 
+// Pairs a vision service with a camera, informing the service about which camera it may use
+message ObstacleDetector {
+  common.v1.ResourceName vision_service = 1;
+  common.v1.ResourceName camera = 2;
+}
+
 message MotionConfiguration {
-  // The name of the vision service(s) that will be used to detect obstacles
-  repeated common.v1.ResourceName vision_services = 1;
+  // Pairs a vision service with a camera, informing the service about which camera it may use
```

Comment:
[nit] Probably don't need to repeat this comment. Instead, you could say "The ObstacleDetectors that will be used for transient obstacle avoidance"

Reply:
Seconded